pom.xml

```diff
@@ -194,7 +194,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>io.jenkins</groupId>
             <artifactId>configuration-as-code</artifactId>
-            <version>1.14</version>
+            <version>1.31</version>
```

Comment:
Is there a reason for this dependency upgrade? Most of your other commits have a note about why some version was changed. Same comment applies for parent pom version.

Reply:
Both upgrades are for staying up-to-date and to receive new features and bug fixes.
I had no specific changes in these dependencies in mind.